index.html

```diff
@@ -8,181 +8,188 @@
-                <img src="img\Unionside.svg">
-            </article>
-            <article class="dropdownHead">
-                <h4>Skills</h4>
-                <img src="img\Unionside.svg">
-            </article>
+                <button class="button">See Project</button>
+            </section>
         </section>
-        <section class="contact">
-            <h1>Contact me</h1>
-            <p class="description">If you have an application you are interested in 
-                developing, a feature that you need built or a project that needs coding. 
-                I’d love to help with it 
-            </p>
-            <form action="https://formspree.io/f/mgeqpzab" method="POST">
-                <input class="single-line-input" type="text" name="name" placeholder="Yeremias" maxlength="30" required>
-                <input class="single-line-input" type="email" name="email" placeholder="natajayanj14@gmail.com" required>
-                <textarea class="comment" placeholder="Write your message here" required></textarea>
-                <button class="button">Get In Touch</button>
-            </form>
+        <section class="presentation" id="aboutContainer">
+            <section class="content" id="about">
+                <h1>About<br>Myself</h1>
+                    <p>
+                        Hello I’m a software developer! I can help 
+                        you build a product , feature or website 
+                        Look through some of my work and experience! 
+                        If you like what you see and have a project you 
+                        need coded, don’t hestiate to contact me.
+                    </p>
+                    <h3>LET’S CONNECT</h3>
+                    <ul class="socialIcon">
+                        <li><a href="#"><img class="icon" src="img\Vector.svg" alt="Tweeter icon"></a></li>
+                        <li><a href="#"><img class="icon" src="img\Linkedin icon.svg" alt="Linkedin icon"></a></li>
+                        <li><a href="#"><img class="icon" src="img\Vector m.svg" alt="M icon"></a></li>
+                        <li><a href="#"><img class="icon" src="img\github icon.svg" alt="github icon"></a></li>
+                        <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="peace icon"></a></li>
+                    </ul>
+                    <button class="button">Get my resume</button>
+            </section>
+            <section class="languages">
+                <article class="dropdownHead" id="open">
+                    <h4>Languages</h4>
+                    <img src="img\Union.svg">
+                </article>
+                <ul class="listOfLanguages">
+                    <li><img src="img\javaicon.svg" alt="javaScript"><p>JavaScript</p></li>
+                    <li><img src="img\htmlicon.svg" alt="HTML"><p>HTML</p></li>
+                    <li><img src="img\cssicon.svg" alt="CSS"><p>CSS</p></li>
+                </ul>
```

Comment:
- [ ] Kindly make the necessary changes to make this part look good on any screen size 👍 

![image](https://user-images.githubusercontent.com/50721479/199977215-f426519a-c6e4-44cc-a467-2b0ac9d0864f.png)